app/connectors/exchange/ExportsDeclarationExchange.scala

```diff
@@ -39,7 +39,6 @@ case class ExportsDeclarationExchange(
   updatedDateTime: Instant,
   sourceId: Option[String],
   `type`: DeclarationType,
-  dispatchLocation: Option[DispatchLocation] = None,
```

Comment:
Since you are changing the contract between services, please update the other service (customs-declare-exports) as well. This field is optional on both ends so I don't expect it to cause problems if they are deployed independently.